CHANGELOG.md

```diff
@@ -1,13 +1,15 @@
 # Changelog
 
 ## Unversioned
-- Major: We now support image thumbnails coming from the link resolver. This feature is off by default and can be enabled in the settings with the "Show link thumbnail" setting. This feature also requires the "Show link info when hovering" setting to be enabled (#1664)
-- Major: Added image upload functionality to i.nuuls.com. This works by dragging and dropping an image into a split, or pasting an image into the text edit field. (#1332)
-- Minor: You can now open the Twitch User Card by middle-mouse clicking a username. (#1669)
-- Minor: Emotes in the emote popup are now sorted in the same order as the tab completion (#1549)
-- Minor: Removed "Online Logs" functionality as services are shut down (#1640)
-- Bugfix: Fix preview on hover not working when Animated emotes options was disabled (#1546)
-- Bugfix: FFZ custom mod badges no longer scale with the emote scale options (#1602)
-- Bugfix: MacOS updater looked for non-existing fields, causing it to always fail the update check (#1642)
+- Major: We now support image thumbnails coming from the link resolver. This feature is off by default and can be enabled in the settings with the "Show link thumbnail" setting. This feature also requires the "Show link info when hovering" setting to be enabled ([#1664](https://github.com/Chatterino/chatterino2/pull/1664))
+- Major: Added image upload functionality to i.nuuls.com with an ability to change upload destination. This works by dragging and dropping an image into a split, or pasting an image into the text edit field. ([#1332](https://github.com/Chatterino/chatterino2/pull/1332), [#1741](https://github.com/Chatterino/chatterino2/pull/1741))
+- Minor: You can now open the Twitch User Card by middle-mouse clicking a username. ([#1669](https://github.com/Chatterino/chatterino2/pull/1669))
+- Minor: User Popup now also includes recent user messages ([#1729](https://github.com/Chatterino/chatterino2/pull/1729))
+- Minor: BetterTTV / FrankerFaceZ emote tooltips now also have emote authors' name ([#1721](https://github.com/Chatterino/chatterino2/pull/1721))
+- Minor: Emotes in the emote popup are now sorted in the same order as the tab completion ([#1549](https://github.com/Chatterino/chatterino2/issues/1549))
+- Minor: Removed "Online Logs" functionality as services are shut down ([#1640](https://github.com/Chatterino/chatterino2/issues/1640))
+- Bugfix: Fix preview on hover not working when Animated emotes options was disabled ([#1546](https://github.com/Chatterino/chatterino2/issues/1546))
+- Bugfix: FFZ custom mod badges no longer scale with the emote scale options ([#1602](https://github.com/Chatterino/chatterino2/pull/1602))
+- Bugfix: MacOS updater looked for non-existing fields, causing it to always fail the update check ([#1642](https://github.com/Chatterino/chatterino2/pull/1642))
```

Comment:
Please undo the link additions here, I rely on GitHub being smart enough to autolink these at some point, and not linking them makes them cleaner to read in a text editor

Reply:
Alright, I reverted those changes.